### PR TITLE
change settings interface

### PR DESCRIPTION
**Pull request by JosiasAurel**
addresses #1383 

```shell
$ mise settings
  List all settings

$ mise settings a
  Get the value of setting with name 'a'

$ mise settings a=b
  Set setting with name 'a' to 'b'
```

### CODE REVIEW

File: src/cli/settings/set.rs [hunk 1]
```diff
@@ -1,133 +0,0 @@
-use eyre::Result;
```
Comment (jdx) [inline src/cli/settings/set.rs:1]: users may still be using the old syntax so they need to be able to still use these old commands

**Comment by camsteffen** [comment]
I think it would be simpler if the setting value is a separate argument instead of using `=`? This is how `git config` works. - `mise settings set x value`

**Comment by jdx** [comment]
@camsteffen I want to support both styles, but this matches how `mise set` works for consistency

**Comment by camsteffen** [comment]
Sorry I meant `mise settings x value`. Would that work?

**Comment by jdx** [comment]
ah yeah, that could be done I think

**Comment by jdx** [comment]
closing due to inactivity, feel free to reopen if you'd like to work on it more

**Pull request closed without merging**